### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by FouadLatrach** [shafayeatsumit/wiseshopper#2]
Hi @shafayeatsumit.
Please help me, i'm new in coding. Can you please explain how can i install this code in a website.
Thanks

**Comment by n8bitsats** [comment]
Yes I also want to contribute to this project but there is no configuration guidance??
